inc/alert.class.php

```diff
@@ -216,6 +216,26 @@ public function rawSearchOptions()
         return $tab;
     }
 
+    // @codingStandardsIgnoreStart
+    public function post_updateItem($history = true)
+    {
+        // @codingStandardsIgnoreEnd
```

Comment:
For ```// @codingStandardsIgnoreStart``` and ```// @codingStandardsIgnoreEnd```

you can add 

```<exclude name="PSR1.Methods.CamelCapsMethodName.NotCamelCaps" />```

to ```.phpcs.xml``` file under ```<rule ref="PSR12">``` node

Reply:
Thanks :+1: [b1fcb7b](https://github.com/pluginsGLPI/news/pull/143/commits/b1fcb7b4290a5ede53e6352bca50e47748517340)